src/SidePanel.tsx

```diff
@@ -30,9 +30,9 @@ export default () => {
                 <Button
                     disabled={!device}
                     variant="primary"
-                    className="w-100"
+                    className="tw-w-full"
                     onClick={() => {
-                        console.dir(device);
+                        // console.dir(device);
```

Comment:
Why do you not remove the console.log?

Reply:
I had actually removed it but for some reason it wasn't committed :/ Doing it now :+1: